frontend/lib/api/form/pollingstation/PollingStationFormController.tsx

```diff
@@ -466,12 +442,9 @@ export function PollingStationFormController({
   return (
     <PollingStationControllerContext.Provider
       value={{
+        error,
         formState,
         values,
-        setValues,
-        loading,
```

Comment:
unsure about a local loading state while the entire form (all sections) are in that same state. 
Can you block navigation while loading for instance?

Reply:
I've moved it back up to the controller in a58728d0c5a46bd293e785118f79f565bd2e6745. Indeed useful when we want to block navigation.